features/constants.py

```diff
@@ -6,3 +6,4 @@
 BUGZILLA_API_KEY = os.getenv("BUGZILLA_API_KEY")
 JIRA_API_KEY = os.getenv("JIRA_API_KEY")
 TIMEOUT = '10'
+TEST_FLAW_CVE_ID = os.getenv("TEST_FLAW_CVE_ID")
```

Comment:
I guess we need several different CVE_IDs for different tests, so how about setting the cve_id as a param?